lib/runtime/runtime.dart

```diff
@@ -103,6 +103,9 @@ class Runtime<T> {
         final context = Context.fromNextInvocation(nextInvocation);
 
         final func = _handlers[context.handler];
+        if(func == null) {
+          throw Exception('No handler with name "${context.handler}" registered in runtime!');
```

Comment:
@Schwusch What do you think if we introduce a custom exception for all runtime errors? Just for consistency.

```dart
class RuntimeException implements Exception {
  String cause;
  RuntimeException(this.cause);
}
```